### DISCUSSION TITLE
Display rank gain/loss after a play

### DISCUSSION THREAD

**Discussion by vatei** [ppy/osu#26627] [Ideas] [answered]
Hello, is it planned to have the rank gain or loss number fading out after a play like in stable? Of course, I can scroll down on the score screen to see the pp and rank gain, but it feels _very_ rewarding to see it pop up after finishing a map without having to scroll all the way down
Example of what I mean because I'm not sure it's clear 😅
![image](https://github.com/ppy/osu/assets/45978249/8251e90b-dacd-42b7-8046-fa43cda1ee05)

Have a nice day!

**Comment by peppy** [comment]
We'll probably add this back somewhere, but not in the absolute jank way that stable did it (the fact that it shows like what you show in that screenshot is actually a *bug* not a feature).

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/issues/26743 tracking here
